### ISSUE TITLE
ExpandableSection - toggle text should be more descriptive in examples

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-react#11394]
**Describe the enhancement or change**
To follow the verbiage from design guidelines ("Informs the user what will be shown when the component is expanded") and to also recommend doing so when there may be multiple toggles on a page, the toggle text should be more descriptive and unique in examples. So instead of "Show more", "Show more basic example content" for the basic example and so on.

This will also provide more unique labeling to the expandable section content, which if there are multiple sections expanded on a page and each toggle just has "Show more", then each region will be labeled as just "Show more".

**Is this request originating from a Red Hat product team? If so, which ones and is there any sort of deadline for this enhancement?**

**Any other information?**

**Comment by Mash707** [comment]
Hi @thatblindgeye I would like to work on this issue. Apart from improving the "Show more" text, should we also include `aria-labels` for each example?

**Comment by thatblindgeye** [comment]
@Mash707 there shouldn't be a need to include `aria-label`, though if you spot an example that you feel needs it definitely let me know
